src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -266,6 +266,9 @@ const processDataPoints = (graphConfig, dataTarget) => {
     // Update the interpolation type
     dataTarget.interpolationType = getInterpolationType(dataTarget.type);
 
+    dataTarget.strokeDasharray =
```

Comment:
By doing this you are no longer using the `style` property.

Reply:
Updated